xdsl/dialects/builtin.py

```diff
@@ -1248,6 +1248,13 @@ def __len__(self) -> int:
         return len(self.data.data) // self.elt_type.size
 
 
+DenseI64ArrayConstraint = ParamAttrConstraint(DenseArrayBase, [i64, BytesAttr])
```

Comment:
Should this be `DenseI64ArrayConstr`? Not sure if we have a naming convention here.

Reply:
I prefer Constr, it's what I've used so far